composer.json

```diff
@@ -56,8 +56,8 @@
     "require": {
         "doctrine/inflector": "1.2.0 as 1.3.1",
         "newfold-labs/wp-module-coming-soon": "^1.1.3",
-        "newfold-labs/wp-module-data": "^2.4.1",
-        "newfold-labs/wp-module-ecommerce": "0.9.2",
+        "newfold-labs/wp-module-data": "^2.4.3",
+        "newfold-labs/wp-module-ecommerce": "1.1.1",
```

Comment:
I think we need to roll this module update back. We had a discussion about it in the last release cycle. This version update assumes the UI update is complete, so we are locked to 0.9.2 until we get the UI update into that specific brand plugin. Not ideal, but probably the safest bet. 

See discussion here: https://github.com/newfold-labs/wp-plugin-crazy-domains/pull/103